src/main/java/blf/configuration/EmissionSettings.java

```diff
@@ -0,0 +1,24 @@
+package blf.configuration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EmissionSettings {
+
+    private static final Map<String, EmissionMode> emissionModeMap = new HashMap<>();
+
+    public static Map<String, EmissionMode> getEmissionModeMap() {
+        emissionModeMap.put("default batching", EmissionMode.DEFAULT_BATCHING);
+        emissionModeMap.put("save batching", EmissionMode.SAVE_BATCHING);
```

Comment:
Save or safe? 🤔

Reply:
True was somehow convinced save batching was the right way to write it (e.g. savegame). Will change it just now!